pkgs/development/tools/templ/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "templ";
+  version = "0.2.476";
+
+  subPackages = [ "cmd/templ" ];
+
+  CGO_ENABLED = 0;
+
+  flags = [
+    "-trimpath"
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-extldflags -static"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "a-h";
+    repo = "templ";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-lgeVfe+9kUxN4XXL7ANiFxtmupZwDaiRFABJIAclyd8=";
+  };
+
+  vendorHash = "sha256-hbXKWWwrlv0w3SxMgPtDBpluvrbjDRGiJ/9QnRKlwCE=";
+
+  tags = [
+    "mysql"
+    "go"
+    "golang"
+    "postgres"
+    "orm"
+    "database"
+    "postgresql"
+    "mssql"
+    "sqlite3"
+    "sqlboiler"
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A language for writing HTML user interfaces in Go";
+    homepage = "https://templ.guide/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ];
```

Comment:
If you're willing to maintain this package, you can add yourself to the maintainers list by

1. Adding yourself to `maintainers/maintainer-list.nix` on a separate commit like `maintainers: add luleyleo`
2. Add yourself to the list here.

Otherwise, feel free to add me (`nevivurn`) as a maintainer.

Reply:
I've added myself to the list now.